common/app/templates/inlineJS/blocking/membershipAccess.scala.js

```diff
@@ -29,13 +30,14 @@
 
             if (identity.isUserLoggedIn()) {
                 ajax({
-                    url: membershipUrl + '/user/me',
+                    url: membersDataApiUrl + '/user-attributes/me/membership',
                     type: 'json',
                     crossOrigin: true,
                     withCredentials: true
                 }).then(function (resp) {
+                    var access = resp.contentAccess
                     // Check the users access matches the content
-                    var canViewContent = (requiresPaidTier) ? !!resp.tier && resp.isPaidTier : !!resp.tier;
+                    var canViewContent = access && (requiresPaidTier) ? !!access.paidMember : !!access.member;
```

Comment:
This took a bit of squinting. Not sure what the brackets around `(requiresPaidTier)` are for. And the `!!` casts to boolean are unnecessary since this is ultimately used in an `if` condition which will implicitly cast to boolean. To me, it would be much clearer like this:
```
access && (requiresPaidTier ? access.paidMember : access.member)
```